src/components/composite/forms/ProfileFormWithContext.tsx

```diff
@@ -26,13 +28,26 @@ export interface ProfileFormValuesType {
   references: Reference[];
   // visuals: Visual2[]; todo: enable when it's time
   tagsets: Tagset[];
+  innovationFlowTemplateID: string;
+}
+
+interface LifecycleTemplateInfo {
+  id: string;
+  title?: string;
+}
+interface LifecycleTemplate {
+  definition: string;
+  id: string;
+  type: LifecycleType;
+  info: LifecycleTemplateInfo;
 }
 
 interface Props {
   context?: Context;
   name?: string;
   nameID?: string;
   tagset?: Tagset;
+  lifecycleTemplates: LifecycleTemplate[] | undefined;
```

Comment:
innovationFlowTemplates --> just the prop name. We leave the name of the type LifecycleTemplate as it is more generic.